vdev/src/commands/meta/compile_vrl_wasm.rs

```diff
@@ -0,0 +1,36 @@
+use std::path::Path;
+use anyhow::Result;
+use clap::Args;
+
+use crate::app;
+
+/// Compiles VRL crates to wasm32-unknown-unknown
+#[derive(Args, Debug)]
+#[command()]
+pub struct Cli {}
+
+impl Cli {
+    pub fn exec(self) -> Result<()> {
+        let vrl_path = Path::new(app::path()).join("lib").join("vrl");
+        // Can't use cargo vdev build because it defaults to Error: command: "cargo" "build" "--no-default-features" "--release" "--features" "default" "--target" "wasm32-unknown-unknown"
+        // Notice the --features "default", some of the crates below don't have default specified
+        // Causing the error: none of the selected packages contains these features: default
+        let args = vec!["build", "--release", "--target", "wasm32-unknown-unknown"];
+        println!("Compiling lib/vrl/compiler to wasm32-unknown-unknown");
+        std::env::set_current_dir(vrl_path.join("compiler"))?;
+        app::exec("cargo", &args, false)?;
```

Comment:
This stanza is run _identically_ four times, just with different subdirectories. I suggest you set that up as a loop over `["compiler", "core", "diagnostic", "parser"]`.